api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java

```diff
@@ -367,5 +369,12 @@ private void addRunLog(final PipelineRun run, final String logMessage, final Str
                 .build();
         runLogManager.saveLog(runLog);
     }
+
+    private void removeUtilizationLevelTags(final PipelineRun run) {
+        Stream.of(ResourceMonitoringManager.UTILIZATION_LEVEL_LOW,
```

Comment:
We also need to persist these changes to db using dao

Reply:
Done